.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    if: "contains(github.event.push.commits[0].message, 'version')"
```

Comment:
not sure if this will do tho, documentation won't show what's inside the payload under commits https://developer.github.com/webhooks/event-payloads/#push but I feel like it's somehow similar to this https://developer.github.com/v3/repos/commits/#list-commits for the `commit` key